plugins/@grouparoo/sailthru/src/lib/client.ts

```diff
@@ -0,0 +1,107 @@
+import { SimpleAppOptions } from "@grouparoo/core";
+import Sailthru from "sailthru-client";
+
+export default class SailthruClient {
+  sailthru: any;
+
+  constructor(options: SimpleAppOptions) {
+    const apiKey = options.apiKey;
+    const apiSecret = options.apiSecret;
+
+    this.sailthru = Sailthru.createSailthruClient(apiKey, apiSecret);
+    //this.sailthru.enableLogging();
```

Comment:
Maybe this choice should be an app or connection option?  Folks might have API trouble in production?